src/components/Header/CalendarUnitListItem.tsx

```
@@ -7,16 +7,15 @@ interface ListItemProps {
 }
 
 export default function CalendarUnitListItem({ calendarUnitItem, onClick: handleClickItem }: ListItemProps) {
-  const { actions } = useMainCalendarStore();
+  const { actions } = useMainCalendarStore();jjkhh
```

ESLint

[error] 'actions' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)